src/Handler/AbstractProcessingHandler.php

```diff
@@ -15,12 +7,19 @@
 use Throwable;
 use WeCodeIn\ErrorHandling\Processor\ProcessorInterface;
 
-trait ProcessableTrait
+abstract class AbstractProcessingHandler extends AbstractHandler
 {
-    /** @var ProcessorInterface[] */
-    private $processors = [];
+    /**
+     * @var ProcessorInterface[]
+     */
+    private $processors;
 
-    protected function setProcessors(ProcessorInterface ...$processors)
+    public function __construct(array $processors = [])
```

Comment:
I prefer ProcessorInterface ...$processors instead of array $processors = [] as you can later when you creating instance of AbstractProcessingHandler implementation pass any iterable easy

eg.
```
$queue = new SplQueue();
$queue->enqueue($processor1);
$queue->enqueue($processor2);

$handler = new ErrorHandler(...$queue);
```